Resources/Private/BackendModule/src/Components/FocusKeywordModuleConfigurationForm.tsx

```diff
@@ -0,0 +1,150 @@
+import PureComponent from "./PureComponent";
+import {connect} from "react-redux";
+import StateInterface from "../Store/StateInterface";
+import {setModuleConfiguration} from "../Store/AppSlice";
+import LimitField from "./ConfigurationForm/LimitField";
+import React from "react";
+import {enumKeys} from "../Util";
+import {FocusKeywordModuleConfiguration, FocusKeywordModuleMode} from "../Model/FocusKeywordModuleConfiguration";
+import PropTypes from "prop-types";
+
+@connect((state: StateInterface) => ({
+    configuration: state.app.moduleConfiguration,
+    initialConfiguration: state.app.initialModuleConfiguration,
+    availableNodeTypeFilters: state.app.availableNodeTypeFilters
+}), (dispatch) => ({
+    updateConfiguration(moduleConfiguration: FocusKeywordModuleConfiguration) {
+        dispatch(setModuleConfiguration({moduleConfiguration}))
+    }
+}))
+export default class FocusKeywordModuleConfigurationForm extends PureComponent {
```

Comment:
FocusKeywordModuleConfigurationForm und AssetModuleConfigurationForm sehen für mich sehr ähnlich aus, und wiederholen immer den selben visuellen Teil, ich denk mit einer rein-visuellen SelectField-komponente (sie in Fusion Monocle) lässt sich viel Code sparen.

Eigenlich müsste ein Configuration-Array für die Filter reichen. Die Aktionen könnten dann in AssetModuleConfigurationForm auch existieren und generisch sein.